internal/pkg/telemetry/telemetry.go

```diff
@@ -43,6 +43,7 @@ type CpuUsage struct {
 var once sync.Once
 var lastSample CpuUsage
 var usageAvg float64
+var WaitGroup sync.WaitGroup
```

Comment:
This var doesn't need to be exported. Should be `waitGroup`. Otherwise LGTM.

Reply:
Addressed.